### PR TITLE
gregorybel: Blukon improvements (2)

### PR DESCRIPTION
**Pull request by gregorybel**
- BT watchdog timer value configurable
- add Libre SN for Blucon
- code optimisation for Blukon Module
- stop resetting bridge_battery and nfc_sensor_age after each initialisation to simply keep current value

successfully tested!

### CODE REVIEW

File: wear/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java [hunk 1]
```diff
@@ -208,8 +208,8 @@ public synchronized static void updateDB() {
     }
 
     public double getDg_mgdl(){
-        if(dg_mgdl != 0) return dg_mgdl;
-        return calculated_value;
+        if(dg_mgdl != 0) return dg_mgdl;//from plugin
```
Comment (jamorham) [inline wear/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java]: These comments are not correct. Display glucose covers much more than just plugins, it handles noise compensation and more. I would suggest removing the comment at least from `dg_mgdl` as it is misleading. `calculated_value` is also not guaranteed to be the original xDrip algorithm either.

**Comment by jamorham** [comment]
This looks fine other than the item I referenced in my inline comment.

File: wear/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java (hunk 1 above)
Comment (gregorybel) [reply wear/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java]: ok, removed!

**Comment by gregorybel** [comment]
@jamorham can you noy hopefully merge it?

**Comment by gregorybel** [comment]
cool! you are welcomed, could you please merge now?

**Pull request merged**